src/plugins/intel_gna/src/gna_graph_tools.hpp

```diff
@@ -86,7 +86,7 @@ inline InferenceEngine::CNNLayerPtr CNNNetPrevLayer(const InferenceEngine::CNNLa
  * @param idx - index in previous layer collection
  * @param layer
  */
-inline InferenceEngine::CNNLayerPtr CNNNetPrevLayer(const InferenceEngine::CNNLayer* layer, int idx = 0) {
+inline InferenceEngine::CNNLayerPtr CNNNetPrevLayer(const InferenceEngine::CNNLayer* layer, size_t idx = 0) {
```

Comment:
ensure that negative value is not used as condition before you change int to size_t